app/assets/stylesheets/pages/_default.scss

```diff
@@ -92,6 +92,10 @@
   }
 }
 
+.__inline{
```

Comment:
This feels like it should belong in a different file. Perhaps a file under `app/assets/stylesheets/helpers/` titled `_utilities.scss`?

Also, perhaps you could rename the class to `inline` instead of `__inline`... Though I'm open to suggestion there.

Reply:
I like both suggestions. I prefer name the css class as `inline`